v4/source/users.yaml

```diff
@@ -8,7 +8,7 @@
         ##### Permissions
         No permission required but user creation can be controlled by server configuration.
       parameters:
-        - name: t
+        - name: iid
```

Comment:
Thanks for the pull requset @Dhiver! We actually do use `t` for invites coming an email and `iid` when it's coming from an invite link. Mind updating this to include both parameters with a short description for each?